src/openrct2/rct1/Tables.h

```diff
@@ -24,7 +24,7 @@ namespace RCT1
     enum class RideType : uint8_t;
 
     colour_t GetColour(colour_t colour);
-    PeepSpriteType GetPeepSpriteType(uint8_t rct1SpriteType);
+    ::PeepSpriteType GetPeepSpriteType(PeepSpriteType rct1SpriteType);
```

Comment:
```suggestion
    ::PeepSpriteType GetPeepSpriteType(::RCT1::PeepSpriteType rct1SpriteType);
```
Should solve make the CI happy